src/main/resources/templates/cypher/list.html

```diff
@@ -18,6 +18,7 @@
 
 <body>
 <div th:insert="fragments/header :: header(cypher)"></div>
+<div class="container">
```

Comment:
Toto si asi nechtel pridat, nebo jo?

Reply:
Sakra, to jsem nechtěl. Ani nevím, jak se to tam vzalo....